### PR TITLE
Move exported classes out of internal package

### LINKED ISSUE #22819: Refactor package names and/or place classes used by MicroProfile Metrics into
Package names of bundles related to Microprofile metrics that contain "internal" are sometimes exported as  specified in their respective `bnd.bnd` files. This is not the desired behavior.

Need to evaluate the packaging of classes relative to the package name and refactor and re-order as necessary.

### PR DESCRIPTION
**Pull request by Channyboy**
#build #libby
fixes #22819

### CODE REVIEW

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build #libby

**Comment by Channyboy** [comment]
Rebuild as the last (and successful) build is 4 days old . #build

**Comment by Channyboy** [comment]
#build #libby

File: dev/io.openliberty.microprofile.faulttolerance.4.0.internal.metrics.5.0/src/io/openliberty/microprofile/faulttolerance40/internal/metrics/integration/MetricRecorderProvider30Metrics50Impl.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
 import com.ibm.ws.microprofile.faulttolerance.spi.TimeoutPolicy;
 import com.ibm.ws.microprofile.faulttolerance.utils.DummyMetricRecorder;
 
-import io.openliberty.microprofile.metrics50.internal.SharedMetricRegistries;
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.faulttolerance.4.0.internal.metrics.5.0/src/io/openliberty/microprofile/faulttolerance40/internal/metrics/integration/MetricRecorderProvider30Metrics50Impl.java:38]: You need to rename the file to match the package.

File: dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/microprofile/metrics50/package-info.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/microprofile/metrics50/package-info.java:2]: This is the old license

File: dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/microprofile/metrics50/rest/package-info.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/microprofile/metrics50/rest/package-info.java:2]: Update the license

**Review by fmhwong — CHANGES_REQUESTED** [review]
Please update license to Eclipse Public License 2.0.

**Comment by Channyboy** [comment]
#build #libby

**Review by fmhwong — APPROVED** [review]
LGTM

**Pull request merged**